addons/website_sale/models/res_config_settings.py

```diff
@@ -52,6 +52,7 @@ class ResConfigSettings(models.TransientModel):
     salesperson_id = fields.Many2one(
         related='website_id.salesperson_id',
         readonly=False,
+        default=lambda self: self.env.ref('base.user_admin').id,
```

Comment:
Why do you set the default on the `res.config.settings` model and not the `website` model ?
This default value is applied every time you open the settings, if the admin is removed from the salesperson field, it'll be applied once again if anybody saves the settings afterwards.